package-lock.json

```diff
@@ -181,7 +181,7 @@
 				},
 				"minimist": {
 					"version": "1.2.0",
-					"resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
+					"resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```

Comment:
Can you revert the changes on this file since it only changed the protocol of the URLs?

Reply:
Strange. I actually try to avoid committing the package-lock. Anyway: I'll revert it.